Monika After Story/game/event-handler.rpy

```diff
@@ -2004,44 +2471,45 @@ init python:
         evhand._lockEventLabel(evlabel, eventdb=eventdb)
 
 
+    @store.mas_utils.deprecated("MASEventList.push")
     def pushEvent(event_label, skipeval=False, notify=False):
```

Comment:
Maybe we could remove all our uses of the deprecated funcs?

Reply:
there's alot and pretty much no places need context yet so we could just change as we modify the surrounding code. We have to keep this as a non-exception deprecation because of submods anyway.

Reply:
I know, it's just we use quite a few deprecated funcs and they're flooding logs. I was thinking about opening a PR to remove all uses of deprecated funcs, so we can leave it until then if you want.

Reply:
yeah lets do that in a subsequent pr